rasa/core/cli/train.py

```diff
@@ -50,6 +50,9 @@ def add_interactive_args(parser):
         default=False,
         action='store_true',
         help="retrain the model immediately based on feedback.")
+    parser.add_argument('--data',
```

Comment:
`--data` could be a bit misleading here, right? Should we call it `--nlu`  as it's done for the NLU training? 
@erohmensing Any wishes?

Reply:
@wochinge I agree that it is misleading but `--nlu` is not possible here since it is already used for the directory of the trained nlu model. But what this parameter refers to is the path to the intent training data file. The rasa nlu training parameter for this file is also called `--data` (see here: [rasa/nlu/train.py#L26](https://github.com/RasaHQ/rasa_nlu/blob/7ca592a2e78380388251adf58236994299488761/rasa/nlu/train.py#L26)) and therefore I thought it would be a good idea to keep it consistent.

Reply:
@Spenhouet I think that when you're training NLU, it makes sense for `--data` to mean NLU data. But when you're training core, you'd expect `--data` to mean core data. I would suggest something like `--nlu_data` or something of the sort

Reply:
@erohmensing `--nlu_data` sounds good to me 👍

Reply:
Awesome, then let's do that 🚀

Reply:
@wochinge I did change the argument naming 🚀